include/jemalloc/internal/hpa_opts.h

```diff
@@ -84,7 +166,13 @@ struct hpa_shard_opts_s {
 	/* min_purge_interval_ms */					\
 	5 * 1000,							\
 	/* experimental_max_purge_nhp */				\
-	-1								\
+	-1,      							\
+	/* size_t purge_threshold */					\
+	PAGE,								\
+	/* min_purge_delay_ms */             				\
+	0,  								\
+	/* hugify_style */                				\
+	hpa_hugify_style_lazy						\
```

Comment:
Is there a reason this should not be determined based on the global THP enabled setting?  I think having a lazy style of promotion is never a good idea when the THP enabled value is `always` or `none`.